lib/galaxy_test/selenium/test_collection_edit.py

```diff
@@ -42,64 +43,58 @@ def test_change_datatype_simple_list(self):
         self.change_datatype_value_and_click_submit(dataValue, dataNew)
         self.check_current_data_value(dataNew)
         self.wait_for_history()
-        self.find_element_by_selector("span.content-title").click()
-        self.wait_for_selector_clickable("div.content-item")
-        self.find_element_by_selector("div.content-item").click()
-        item = self.history_panel_item_component(hid=1)
-        item.datatype.wait_for_visible()
-        assert self.find_element_by_selector("span.datatype > span").text == dataNew
-
-    def create_simple_list_collection(self):
-        self.perform_upload(self.get_filename("1.fasta"))
-        self._wait_for_and_select([1])
-        self._collection_dropdown("build list")
-        self.collection_builder_set_name("my cool list")
-        self.screenshot("collection_builder_list")
-        self.collection_builder_create()
-        self._wait_for_hid_visible(2)
 
-    def create_simple_list_collection_txt(self):
-        self.perform_upload(self.get_filename("1.txt"))
+        @retry_assertion_during_transitions
+        def check_dataset_converted():
```

Comment:
```suggestion
        def check_dataset_changed():
```
the conversion would always create a new HDA, so I first thought `self.history_panel_item_component(hid=1)` was the problem